code/modules/surgery/advanced/lobotomy.dm

```diff
@@ -0,0 +1,66 @@
+/obj/item/disk/surgery/lobotomy
+	desc = "The disk provides instructions on how to perform a lobotomy, to cure the most resilient brain ailments."
+	surgeries = list(/datum/surgery/advanced/lobotomy)
+
+/datum/surgery/advanced/lobotomy
+	name = "lobotomy"
+	steps = list(
+	/datum/surgery_step/incise,
+	/datum/surgery_step/retract_skin,
+	/datum/surgery_step/saw,
+	/datum/surgery_step/clamp_bleeders,
+	/datum/surgery_step/lobotomize,
+	/datum/surgery_step/close)
+
+	species = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list("head")
+	requires_bodypart_type = 0
+
+/datum/surgery/advanced/lobotomy/can_start(mob/user, mob/living/carbon/target)
+	if(!..())
+		return FALSE
+	var/obj/item/organ/brain/B = target.getorganslot(ORGAN_SLOT_BRAIN)
+	if(!B)
+		return FALSE
+	return TRUE
+
+/datum/surgery_step/lobotomize
+	name = "perform lobotomy"
+	implements = list(/obj/item/scalpel = 85, /obj/item/melee/transforming/energy/sword = 55, /obj/item/kitchen/knife = 35,
+		/obj/item/shard = 25, /obj/item = 20)
+	time = 100
+
+/datum/surgery_step/lobotomize/tool_check(mob/user, obj/item/tool)
+	if(implement_type == /obj/item && !tool.is_sharp())
+		return FALSE
+	return TRUE
+
+/datum/surgery_step/lobotomize/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("[user] begins to cut a piece of [target]'s brain.", "<span class='notice'>You begin to cut a piece of [target]'s brain...</span>")
+
+/datum/surgery_step/lobotomize/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("[user] successfully lobotomizes [target]!", "<span class='notice'>You succeed in lobotomizing [target].</span>")
+	target.cure_all_traumas(TRAUMA_RESILIENCE_LOBOTOMY)
+	switch(rand(1,4))//Now let's see what hopefully-not-important part of the brain we cut off
+		if(1)
+			target.gain_trauma_type(BRAIN_TRAUMA_MILD, TRAUMA_RESILIENCE_MAGIC)
```

Comment:
so is this never fixable or only fixable through cloning?

Reply:
Either cloning or full heals; i think legion hearts and changeling revives should be able to do that.